docs/global.css

```diff
@@ -116,10 +34,6 @@
 .root {
 }
 
-.sectionBreaker {
-  margin: 30px 0px;
-  width: 80%;
-  border: 1px solid #e9e9e9;
-  height: 0px;
-  margin: 20px auto 20px;
+:global pre {
+  font-size: 0.95em;
```

Comment:
Why `em` here and not elsewhere? There is a [discussion to be had](https://j.eremy.net/confused-about-rem-and-em/) as to whether we should be using (r)em's everywhere though.

Reply:
95% of the font size of the parent yes. My opinion isn't that strong either, just seems inconsistent.